index.html

```diff
@@ -4,19 +4,19 @@
 <head>
   <title>Dustin Yelland</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="reset.css" rel="stylesheet">
-  <link href="base.css" rel="stylesheet">
-  <link href="modules.css" rel="stylesheet">
+  <link href="styles/reset.css" rel="stylesheet">
+  <link href="styles/base.css" rel="stylesheet">
+  <link href="styles/modules.css" rel="stylesheet">
```

Comment:
Yay! fix up your selectors that I mentioned in your day 2 PR and resubmit this for your day 2 assignment and it'll get you 10/10 on day 2!